dlib/geometry/vector_abstract.h

```diff
@@ -456,6 +456,21 @@ namespace dlib
               false if not.
     !*/
 
+// ----------------------------------------------------------------------------------------
+
+    template <typename T>
+    std::enable_if_t<std::is_same<T, point>::value || std::is_same<T, dpoint>::value, std::vector<T>>
+    find_convex_hull(
+        std::vector<T>& points
+    );
+    /*!
```

Comment:
```suggestion
    std::vector<T> find_convex_hull(
        std::vector<T>& points
    );
    /*!
        requires
            - T == dlib::point or dlib::dpoint
```
A little more readable :)